VocaDbModel/Domain/Songs/ISongList.cs

```diff
@@ -0,0 +1,13 @@
+using VocaDb.Model.Domain.Users;
+
+namespace VocaDb.Model.Domain.Songs {
+
+	public interface ISongList {
+
+		IUser Author { get; }
+
+		bool FeaturedList { get; }
+
+	}

```

Comment:
I added ISongList so that EntryPermissionManager.CanEdit can be called from SongListDetailsViewModel.